pilot/pkg/model/push_context.go

```diff
@@ -2177,6 +2177,24 @@ func (ps *PushContext) EnvoyFilters(proxy *Proxy) *EnvoyFilterWrapper {
 		matchedEnvoyFilters = append(matchedEnvoyFilters, matched...)
 	}
 
+	sort.Slice(matchedEnvoyFilters, func(i, j int) bool {
```

Comment:
BTW, we may not need to sort envoyfilters in `initEnvoyFilters` any more

Reply:
Yes. But I just left it if envoy filters are used via some other way other than calling EnvoyFilters function for any other purpose

Reply:
you can write 

```
if ifilter.Priority < jfilter.Priority {
	return true
}
```

Reply:
yeah but I thought sliceStable might perform slow - any case changed. PTAL